packages/app_center/lib/src/snapd/snap_search.dart

```diff
@@ -43,6 +43,16 @@ final snapSearchProvider =
               .toList() ??
           [],
     );
+  } else if (searchParameters.category == SnapCategoryEnum.gameDev ||
```

Comment:
This is very ugly, maybe change the enum for a gaming bundles enum and just test against obj.runtimeType?

Reply:
this introduces unwanted complications so maybe leave it as is